opeanapi-spec.yaml

```diff
@@ -0,0 +1,122 @@
+          Thing Descriptions and System Descriptions should be passed as they-are (i.e., as JSON objects)
+        required: true
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - type: object
+                  properties:
+                    puml:
+                      type: string
+                    tds:
+                      type: object
+                - type: object
+                  properties:
+                    sd:
+                      type: object
+            examples:
+              Sequence Diagram To System Description:
+                value:
+                  puml: "%SequenceDiagramAsURIEncodedString%"
+                  tds: {}
+              System Description To Sequence Diagram:
+                value:
+                  sd: {}
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: object
+                    properties:
+                      success:
+                        type: boolean
+                      code:
+                        type: object
+                        properties:
+                          ts:
+                            type: string
+                          tsconfig:
+                            type: string
+                          js:
+                            type: string
+                      sd:
+                        type: string
+                  - type: object
+                    properties:
+                      success:
+                        type: boolean
+                      code:
+                        type: object
+                        properties:
+                          ts:
+                            type: string
+                          tsconfig:
+                            type: string
+                          js:
+                            type: string
+                      puml:
+                        type: string
+                      tds:
+                        type: string
+              examples:
+                Sequence Diagram To System Description:
+                  value:
+                    success: true
+                    code:
+                      ts: "http://127.0.0.1:3000/54a3f07fc9135e44/code/54a3f07fc9135e44.ts"
+                      tsconfig: "http://127.0.0.1:3000/54a3f07fc9135e44/tsconfig.json"
+                      js: "http://127.0.0.1:3000/54a3f07fc9135e44/code/54a3f07fc9135e44_index.js"
+                    sd: "http://127.0.0.1:3000/54a3f07fc9135e44/sds/54a3f07fc9135e44.json"
+                System Description To Sequence Diagram:
+                  value:
+                    success: true
+                    code:
+                      ts: "http://127.0.0.1:3000/d7db70f9a3cb4bd1/code/d7db70f9a3cb4bd1.ts"
```

Comment:
shouldn't this be a variable?

Reply:
Sorry, what kind of variable do you mean?